content/docs/details/accessing-aws-resources/index.md

```diff
@@ -87,7 +89,7 @@ You should see a page like this
         },
         "StringLike": {
           "aws:PrincipalArn": [
-            "<ROADIE BACKEND ROLE ARN>"
+            "*<ROADIE BACKEND ROLE>*"
```

Comment:
Maybe could be worth mentioning why the starts (we autogenerate the ending of the role or something?)

Reply:
Like in the note below but an actual example